packages/runtime-core/src/scheduler.ts

```diff
@@ -262,7 +261,7 @@ function checkRecursiveUpdates(seen: CountMap, fn: SchedulerJob) {
     if (count > RECURSION_LIMIT) {
       const instance = fn.ownerInstance
       const componentName = instance && getComponentName(instance.type)
-      warn(
+      throw new Error(
```

Comment:
Use `console.error` instead. `throw` will break and stop the context.

Reply:
so, maybe should using handleError?
https://github.com/vuejs/core/blob/76b8573e2453ac7f63316be161e000086e4b4c4a/packages/runtime-core/src/errorHandling.ts#L100